planning/behavior_path_planner/include/behavior_path_planner/scene_module/start_planner/start_planner_module.hpp

```diff
@@ -59,10 +59,9 @@ struct PullOutStatus
   PlannerType planner_type{PlannerType::NONE};
   PathWithLaneId backward_path{};
   lanelet::ConstLanelets pull_out_lanes{};
-  bool is_safe_static_objects{false};   // current path is safe against static objects
+  bool found_pull_over_path{false};     // current path is safe against static objects
```

Comment:
```suggestion
  bool found_pull_out_path{false};     // current path is safe against static objects
```

and oter parts too

Reply:
oh...
Thanks!

Reply:
changed in https://github.com/autowarefoundation/autoware.universe/pull/5390/commits/3888e187a9408421827fe805900c915960414e5f